### PR TITLE
ZK-4619: The radiogroup of cloned radio is not updated

### PR DESCRIPTION
**Pull request by b952426**
ZK-4619: The radiogroup of cloned radio is not updated

### CODE REVIEW

File: zul/src/org/zkoss/zul/Radiogroup.java [hunk 1]
```diff
@@ -701,9 +702,28 @@ public Object clone() {
...
+			clone.setSelectedIndex(-1);
+		}
 		return clone;
 	}
 
+	private void getRadio(Component comp) {
+		if (_interns == null)
+			_interns = new LinkedList<Radio>();
+		for (Component c : comp.getChildren()) {
+			if(c instanceof Radio) {
```
Comment (scribetw) [inline zul/src/org/zkoss/zul/Radiogroup.java:719]: if (

File: zul/src/org/zkoss/zul/Radiogroup.java [hunk 2]
```diff
@@ -701,9 +702,28 @@ public Object clone() {
 			clone._dataListener = null;
 			clone.initDataListener();
 		}
+		getRadio(clone);
```
Comment (scribetw) [inline zul/src/org/zkoss/zul/Radiogroup.java:705]: Fix Radio instead

**Review by scribetw — CHANGES_REQUESTED** [review]

**Review by scribetw — APPROVED** [review]

**Review by plijyfes — APPROVED** [review]

**Comment by plijyfes** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**